src/Phoenix/PhoenixClient.ts

```diff
@@ -101,19 +130,24 @@ export class PhoenixClient {
     }
   }
 
-  private async getContainerHealth(delayMs: number, containerData: { forwardingId: string; dbAccountName: string }) {
+  private async getContainerHealth(delayMs: number, containerData: IContainerData) {
+    const containerInfo = await this.getContainerStatusAsync(containerData);
+    useNotebook.getState().setContainerStatus(containerInfo);
+    if (useNotebook.getState().containerStatus?.status === ContainerStatusType.Active) {
+      this.scheduleContainerHeartbeat(delayMs, containerData);
+    }
+  }
+
+  public async IsDbAcountWhitelisted() {
```

Comment:
Unlike C# which is PascalCase, typescript is camelCase. Add return type.
```suggestion
  public async isDbAcountWhitelisted(): Promise<boolean> {
```